core/src/main/java/org/elasticsearch/script/ScriptService.java

```diff
@@ -86,6 +88,8 @@
         Setting.boolSetting("script.auto_reload_enabled", true, Property.NodeScope);
     public static final Setting<Integer> SCRIPT_MAX_SIZE_IN_BYTES =
         Setting.intSetting("script.max_size_in_bytes", 65535, Property.NodeScope);
+    public static final Setting<Integer> SCRIPT_MAX_COMPILATIONS_PER_MINUTE =
+        Setting.intSetting("script.max_compilations_per_minute", 15, Property.Dynamic, Property.NodeScope);
```

Comment:
Right. I figure we should have a test for it or explicitly make the min for that parameter `1`. We have other ways to turn off dynamic scripts if we want so maybe it'd be better not to have two ways to do it.